### DISCUSSION TITLE
Error message in Chinese (I think)

### DISCUSSION THREAD

**Discussion by ArturSPires** [ImageMagick/ImageMagick#4469] [Digital Image Processing]
Good morning.

I am trying to find out what is wrong with some TIFF files, and when I try to open them in IMDisplay I get an error message, but it appears to be in Chinese. I tried it in Windows 7 and Windows 10 (both 64 bit versions), with the 8 and 16 versions of ImageMagick, 32 and 64 bits versions and even tried an older version, I always get the message below.

Does anyone know how can I fix whatever is causing it?
Thanks in advance.

![IMDisplay error](https://user-images.githubusercontent.com/94108919/141275492-a113a59b-3356-48d6-af21-9dd912272b71.jpg)

**Comment by dlemstra** [comment]
This is probably an UTF8 message that is not displayed properly. We will need to investigate this, how can we reproduce this issue?

**Reply by ArturSPires** [reply]
I changed the image (I know what is causing the error, it's a non-standard tag) in the link below so it produces the error in IMDisplay.

I hope the link works.

https://1drv.ms/u/s!AjzCGsAUz9QGg3gzbFrkrNUfzv-c?e=odzF8r

**Comment by dlemstra** [comment]
Thanks for reporting this. This will be fixed in the next release.
